webinar/routing/marketing-outline.md

```diff
@@ -0,0 +1,23 @@
+# Exploring Angular 2 Routes
+
+One of the largest changes in Angular 2, has been the full adoption of component based architecture, and this has carried over into Angular 2's approach to routing with the new component router.
```

Comment:
One of the largest changes in Angular 2 has been the full adoption of component-based architecture. This has carried over into Angular 2's approach to routing with the new component router.